roles/ciao-common/tasks/main.yml

```diff
@@ -18,8 +18,13 @@
   - name: Create certificate directory
     file: path=/etc/pki/ciao state=directory
 
-  - name: Create /var/lib/ciao
-    file: path=/var/lib/ciao state=directory owner=ciao group=ciao
+  - name: Create /var/lib/ciao tree
```

Comment:
the cleanup deletes /var/lib/ciao tree

When i want to run the playbook again, ciao fails because it cannot create the database under /var/lib/datastore folder.

So ciao-common creates the directory structure before configuring ciao.